stl/inc/source_location

```diff
@@ -25,7 +25,11 @@ _STD_BEGIN
 _EXPORT_STD struct source_location {
     _NODISCARD static consteval source_location current(const uint_least32_t _Line_ = __builtin_LINE(),
         const uint_least32_t _Column_ = __builtin_COLUMN(), const char* const _File_ = __builtin_FILE(),
+#if defined(__clang__) || defined(__EDG__) // TRANSITION, DevCom-10199227 and LLVM-58951
         const char* const _Function_ = __builtin_FUNCTION()) noexcept {
+#else // ^^^ workaround no workaround vvv

```

Comment:
```suggestion
#else // ^^^ workaround / no workaround vvv
```

Reply:
Urgh saw this too late again lol. 

I thought it was double because 
1. That's what your comment on Discord had
2. I double checked and saw
https://github.com/microsoft/STL/blob/009881898c2112f92ab7ee743eadb148c01329c9/stl/inc/algorithm#L8126
But it looks like far less usage of that. Could be a good first issue to be consistent.

Reply:
Oh, I might copy from there.
Yes, change to `/` can be a good first issue.
There are some `workaround // workaround vvv` below (without `no`)

Reply:
> There are some workaround // workaround vvv below (without no)

Yeah I ballsed up copying and pasting... I've batched your suggestions; just waiting for CI to finish,